edk2toollib/uefi/telem/parser_main.py

```diff
@@ -0,0 +1,660 @@
+# @file parser_main.py
+# TODO: Write Readme and short description
```

Comment:
Maybe leave out the TODO or complete it :)

Reply:
Right now, this PR is mostly for feedback. I'm sure the tool has a ways to go before being "done"  - especially if it's being checked into pytools. The TODOs will iteratively disappear :)